test/zdtm/static/shmemfd.c

```diff
@@ -0,0 +1,111 @@
+#include <fcntl.h>
+#include <linux/memfd.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/syscall.h>
+#include <sys/types.h>
+#include <sys/vfs.h>
+#include <unistd.h>
+#include <sys/mman.h>
```

Comment:
Do we really need all these headers?

#include <linux/memfd.h>
#include <stdint.h>
#include <sys/stat.h>
#include <sys/syscall.h>
#include <sys/types.h>

Above ones look not required at all.

Also it looks like the ones below come from zdtmtst.h:

#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <unistd.h>